components/Section/Section.css

```diff
@@ -0,0 +1,18 @@
+.Section {
+    height: 80px;
+    border: 3px solid black;
+    border-bottom: 0px;
+}
+.Section__header {
+    margin-left: 20px;
+    text-align: center;
+    font-size: 35px;
+}
+.Section2 {
+    border-bottom: 3px solid black;
+    height: 100%;
+}
+Section__header-lower {
```

Comment:
This line is grey'd out for me and seems that it is not getting implemented here. I took a closer look with the Inspect Tools are saw that this has the same CSS for h1{}. You may want to take a closer look at how you are calling this element! I think the answer lies in your HTML file 😄